code/turf/turf.dm

```diff
@@ -759,6 +759,10 @@
 				W.update_icon()
 	return wall
 
+/turf/proc/is_sanctuary()
+  var/area/AR = src.loc
+  return AR.sanctuary
```

Comment:
I think a more elegant way of doing this would be a global(?) proc that goes something like
```
/proc/givemeanicename(atom/X)
 var/area/AR = get_area(X)
 . = AR.sanctuary
```
Which should work on things without getting the turf first. I'm not at my coding setup though, so no guarantees that works. :P

Reply:
Pali on Discord:
> there's an argument to be made for both version (iirc I suggested both variants in the original post in this channel)
define:
 + a tiny bit faster
 - inflexible if a turf wants to override its sanctity
 - uses the : operator which makes it so you won't find the define if you do Find All References on sanctuary etc.
proc:
 - a tiny bit slower (overhead from calling the proc mostly)
 + a turf type can override the default behaviour (i.e. you could have a material which makes a floor tile a sanctuary for example)
I personally would prefer the proc variant unless it's clear that it's called a lot which most likely isn't the case but the define variant is also ok